openstack/swift/templates/_utils.tpl

```diff
@@ -114,14 +114,16 @@ checksum/object.ring: {{ include "swift/templates/object-ring.yaml" . | sha256su
 {{- /**********************************************************************************/ -}}
 When passed via `helm upgrade --set`, the image_version is misinterpreted as a float64. So special care is needed to render it correctly.
 {{- define "swift_image" -}}
-  {{- if typeIs "string" .Values.image_version -}}
-    {{ required "This release should be installed by the deployment pipeline!" "" }}
+  {{- if and (typeIs "string" .Values.image_version) (ne .Values.image_version "DEFINED_BY_PIPELINE") -}}
```

Comment:
We don't need the typeIs anymore in this case.

Reply:
Done